Gruntfile.js

```diff
@@ -227,83 +193,61 @@ module.exports = function(grunt) {
                 src: 'build/test-coverage/lcov/**/*.info',
             }
         },
-        // file concatenation
-        concat: {
-            options: {
-                banner: cssFileHeader,
-            },
-            // css for the "page" template
-            pageCss: {
-                src: pageCssFiles,
-                dest: kbaseStaticDir + 'css/page_concat.css',
-                nonull: true,
-            },
-            // a single file containing all the KBase css!
-            allCss: {
-                src: allCssFiles,
-                dest: kbaseStaticDir + 'css/all_concat.css',
-                nonull: true,
-            }
-        },
 
         // check the source css files are not completely horrendous
+        // config to be updated for scss files
         stylelint: {
             options: stylelintOptions,
-            kbaseCss: {
-                src: staticFilesNoConcat,
-            },
+            concat: {
+                src: kbaseStaticDir + 'css/*.css',
+            }
         },
 
         // Run CSS / SCSS-related tasks
-        // these all modify files in place
+        // these files are modified in place
         postcss: {
-            // clean up the files in kbase-extension/static/kbase/(css|custom)/*.css
-            kbaseCss: {
-                options: cleanExistingCssOptions,
-                src: staticFilesNoConcat,
-            },
             // autoprefix and minify the concatenated css files
             concat: {
                 options: {
                     processors: [
                         // add vendor prefixes
                         require('autoprefixer')(),
-                        // add minification later
-                        // require('cssnano')([
-                        //     "default",
-                        //     {
-                        //         "normalizeWhitespace": {
-                        //             "exclude": true
-                        //         },
-                        //     }
-                        // ]),
+                        // minify
+                        require('cssnano')([
+                            "default",
+                            {
+                                "normalizeWhitespace": {
+                                    "exclude": true
+                                },
+                            }
+                        ]),
                     ],
                 },
                 src: [
                     'kbase-extension/static/kbase/css/*_concat.css',
+                    'kbase-extension/static/kbase/css/appCell.css',
+                    'kbase-extension/static/kbase/css/editorCell.css'
```

Comment:
these files are now produced by scss compilation. Ideally they'd all be merged into one big file... possibly in a future PR.